packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceExecutionState.ts

```diff
@@ -610,30 +610,21 @@ export abstract class ServicePureExecutionState extends ServiceExecutionState {
   }
 
   getExecutionQuery(): RawLambda {
```

Comment:
is there a reason we still need this method?

Reply:
<img width="569" alt="Screenshot 2022-11-02 at 11 41 09 AM" src="https://user-images.githubusercontent.com/73408381/199534346-d4718304-8cfe-411a-85ff-2157fceae12e.png">
you wanna it to be inline?